### PR TITLE
Swaps stack in favor of image

### PR DESCRIPTION
**Pull request by bg451**

### CODE REVIEW

**Comment by bg451** [comment]
ping @treeder

**Comment by treeder** [comment]
The command format doesn't use --image, see https://github.com/iron-io/ironcli for info. `iron worker upload [--zip hello.zip] NAME DOCKER_IMAGE [COMMAND]`

**Comment by bg451** [comment]
@treeder sorry about that, fixed.

**Comment by treeder** [comment]
Need to deploy first, then we can merge this.

**Comment by treeder** [comment]
cc @rkononov

**Comment by bg451** [comment]
sounds good

**Comment by rkononov** [comment]
Ticket in ops repo, need to wait a bit until we will make new snapshot

File: worker/beta/getting_started/index.md [hunk 1]
```diff
@@ -79,10 +79,10 @@ zip -r helloworld.zip .
 Then upload it:
 
 {% highlight bash %}
-iron worker upload --stack ruby-2.1 helloworld.zip ruby helloworld.rb
+iron worker upload --zip helloworld.zip iron/images:ruby-2.1 ruby helloworld.rb
```
Comment (treeder) [inline worker/beta/getting_started/index.md:82]: Need --name now too (required).

File: worker/beta/getting_started/index.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ zip -r myworkername.zip .
 Then upload the zip you just created:
 
 {% highlight bash %}
-iron worker upload --stack MY_STACK myworkername.zip MY_PROGRAM
+iron worker upload --zip myworkername.zip iron/images:MY_STACK MY_PROGRAM
```
Comment (treeder) [inline worker/beta/getting_started/index.md:133]: --name.

**Comment by bg451** [comment]
Docs already updated with this: https://github.com/iron-io/docs/commit/f53a73755f9c72bf33853990b260e718f68fecb0

Gonna close this pr

File: worker/beta/getting_started/index.md [hunk 3]
```diff
@@ -79,10 +79,10 @@ zip -r helloworld.zip .
 Then upload it:
 
 {% highlight bash %}
-iron worker upload --stack ruby-2.1 helloworld.zip ruby helloworld.rb
+iron worker upload --zip helloworld.zip iron/images:ruby-2.1 ruby helloworld.rb
 {% endhighlight %}
 
-Notice the --stack parameter is the same as the Docker container we used above.
+Notice that we use the same docker image as above.
```
Comment (treeder) [inline worker/beta/getting_started/index.md:85]: This could still go in if it's not in already?

**Pull request closed without merging**